.gitignore

```diff
@@ -26,7 +26,6 @@ C:\\nppdf32Log\\debuglog.txt
 .zeus.sock
 .deploy
 .s3cfg
-.tool-versions
```

Comment:
I don't want us to need to go through hoops to use something different than asdf.

My suggestion is to keep this ignored. But create a file `.tool-versions.default` or  `.tool-versions.example`.

If one wants to stay on the default, then `ln -s .tool-versions.default .tool-versions` will do the job.

Reply:
If `.tool-versions` is used only by `asdf` and we're using as default version manager, what's the matter in leaving it included? Won't other tools such as rbenv work if this file is present?

Reply:
For example if I use asdf for ruby but install node from other channels, it may mess-things up. Changing the file locallly for whatever reason to try different versions will require care not to add this to commits. Switching branches, rebasing, etc. with a custom tools versions might be annoying as git will complain that there is a modified file.

For me this is local setup of everybody and I prefer the `.sample` approach. If everybody wants to have it by default, then fine. But I don't like it.

Reply:
I get that, however, we should have a way to enforce Ruby and Node versions and I think this tool-versions + asdf strategy is an easy win. I don't mind the `.sample` approach, in fact it is like that already... missing any mention about it in the docs though 🙄

Reply:
We had some place where we fail in case ruby is not the right version. I guess it would also be possible for node.

Reply:
I don't see it but we can add such check somewhere.

Reply:
I reverted to `.sample` file, resolving conversation.

Reply:
if we are back to `.sample`, then we should still ignore `.tool-versions`.